cfgov/unprocessed/css/enhancements/layout-2-1.less

```diff
@@ -56,7 +56,9 @@
     // Desktop
     .respond-to-min( @bp-med-min, {
       .u-layout-grid_wrapper {
-          grid-auto-columns: 2fr 1fr;
+          // Numbers below are 870 split into three parts
+          // 870 is the 900px breakpoint - 30px of gutters
+          grid-auto-columns: minmax(580px, 2fr) minmax(290, 1fr);
```

Comment:
Looks like `px` in `290` is missing. I'm trying to grok why this is different from 2fr/1fr, since 580px/290px is the same ratio. It is saying the columns will not be smaller than 580px/290px (870px) but may be larger, correct? Why do we need this when it is inside the `@bp-med-min` (901px) media query?

Reply:
This specifically sets the minimum value that each column can be, meaning if the OTHER column is too big, it won't steal all the space normally given to this column.

This only matters when there are items that provide their own layout dimensions, like `img`s or `pre`s or `table`s, but as these can be added arbitrarily in wagtail, we need the grid to account for them.